src/_lib/modules/toasts/toasts.service.ts

```diff
@@ -3,13 +3,14 @@ import { DOCUMENT } from '@angular/common';
 import { ComponentPortal, DomPortalOutlet } from '@angular/cdk/portal';
 import { ToastsComponent } from './toasts.component';
 import { IToast } from './IToast';
+import { Toast } from './Toast';
 
 @Injectable()
 export class Toaster {
 
     private _toastPortal: ComponentPortal<ToastsComponent>;
     private _outlet: DomPortalOutlet;
-    private _toastComponentRef: ComponentRef<ToastsComponent>
+    private _toastComponentRef: ComponentRef<ToastsComponent>;
```

Comment:
This is an existing problem, but could you rename this property to `_toastsComponentRef` please